src/sage/rings/polynomial/multi_polynomial.pyx

```diff
@@ -1980,6 +1980,10 @@ cdef class MPolynomial(CommutativePolynomial):
         Given an ideal `I = (f_1,\dots,f_r)` that contains ``self``,
         find `s_1,\dots,s_r` such that ``self`` `= s_1 f_1 + ... + s_r f_r`.
 
+        INPUT:
+
+        - ``I`` -- an ideal in ``self.parent()``, or tuple of generators of that ideal
```

Comment:
no functionality change, just guarantee a previously only implementation detail in the external API to be used by the new method.